tslint.json

```diff
@@ -61,6 +62,28 @@
     "template-no-negated-async": true,
     "use-lifecycle-interface": true,
     "import-blacklist": [true, "@dynatrace/barista-components"],
+
+    "nx-enforce-module-boundaries": [
+      true,
+      {
+        "depConstraints": [
+          {
+            "sourceTag": "scope:components",
+            "onlyDependOnLibsWithTags": ["scope:components", "scope:testing"]
+          },
+          {
+            "sourceTag": "type:app",
+            "onlyDependOnLibsWithTags": ["type:library"]
+          },
+          {
+            "sourceTag": "scope:examples",
+            "onlyDependOnLibsWithTags": ["scope:components"]
+          }
+        ]
+      }
+    ],
+
+    // Custom Rules
```

Comment:
Does that work? I thought comments are not allowed in json files.

Reply:
in tslint it is allowed but in my opinion, they should be avoided.